src/main/kotlin/com/vaadin/gradle/VaadinPrepareNodeTask.kt

```diff
@@ -0,0 +1,87 @@
+package com.vaadin.gradle
+
+import com.moowork.gradle.node.NodeExtension
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import java.lang.IllegalStateException
+
+/**
+ * Copies the node.js distro downloaded by the com.github.node-gradle.node plugin
+ * to the node/ folder which Vaadin expects.
+ *
+ * You need to manually include the com.github.node-gradle.node plugin into your build file
```

Comment:
It is bit unclear for me now, is this needed or not? If needed, then we should mention that in README.md as well.